blank-typescript/package.json

```diff
@@ -0,0 +1,23 @@
+{
+    "name": "@daileytj/pxb-react-native-template-blank-typescript",
```

Comment:
Watch these...we will need to change the name before we merge these into dev

Reply:
Definitely aware, just needed these to be published to test the cli stuff. I'll make sure these are changed before merging to dev.